### ISSUE TITLE
File name of training images 

### ISSUE THREAD

**Issue by niddal-imam** [Holmeyoung/crnn_pytorch#7]
Hi,

Is it okay to name images' files as follow:
image: /Volumes/EXTERNAL/Models/crnn_chinese_characters_rec-master/to_lmdb/train_images/almofdal_11.jpg
label: المفضل

I wrote the label name of the image in English, and I have wrote the label of the image file in data file .txt in Arabic. Actually I was able to convert the files to lmdb, but when I train the model, it does not print out number of epochs and loss. It just show below info for some minutes and stoped.
CRNN(
  (cnn): Sequential(
    (conv0): Conv2d(1, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu0): ReLU(inplace)
    (pooling0): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
    (conv1): Conv2d(64, 128, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu1): ReLU(inplace)
    (pooling1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
    (conv2): Conv2d(128, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm2): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu2): ReLU(inplace)
    (conv3): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu3): ReLU(inplace)
    (pooling2): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), padding=(0, 1), dilation=1, ceil_mode=False)
    (conv4): Conv2d(256, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm4): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu4): ReLU(inplace)
    (conv5): Conv2d(512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu5): ReLU(inplace)
    (pooling3): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), padding=(0, 1), dilation=1, ceil_mode=False)
    (conv6): Conv2d(512, 512, kernel_size=(2, 2), stride=(1, 1))
    (batchnorm6): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (relu6): ReLU(inplace)
  )
  (rnn): Sequential(
    (0): BidirectionalLSTM(
      (rnn): LSTM(512, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=256, bias=True)
    )
    (1): BidirectionalLSTM(
      (rnn): LSTM(256, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=71, bias=True)
    )
  )
)

**Comment by Holmeyoung** [comment]
Hi, check the file size of your lmdb file. I guess you didn’t access the image successfully. The `txt` file should looks like
```
absolute/path/to/your/image/1.jpg
المفضل
```
If there are still problems, you can tell me.

**Comment by niddal-imam** [comment]
I checked the lmdv files and they are 100 KB and 9 KB for training and 70 KB and 8 KB for val. They are small because I am using small data (100 sample). My txt files look like your example.

/Volumes/EXTERNAL/Models/crnn_pytorch-master/to_lmdb/val_images/img_almofdalh_11.jpg
المفضل

**Comment by Holmeyoung** [comment]
Hi, check the following params in `params.py`
```python
# training process
displayInterval = 100 # interval to be print the train loss
valInterval = 1000 # interval to val the model loss and accuray
saveInterval = 1000 # interval to save model
n_test_disp = 10 # number of samples to display when val the model
```
Look at the comment and you will know the reason is that your training sample amount didn’t reach the `displayInterval` `valInterval`...so change them smaller for example `10` is ok for you

**Comment by niddal-imam** [comment]
Thank you for your reply, and sorry for asking so many question. I made the changes that you suggest, but still does not print the loss. It just run for 15 - 20 mins and stop. I may need more data, right?

**Comment by Holmeyoung** [comment]
Hi, it doesn’t matter. 
1. You really need more data, maybe `100000` sample for `1000` classes.

2. But even 100 samples should print loss either. Change the `batchSize` in `params.py` to `1` and see what will happen. If there is still nothing printed, it’s not the sample problem.

**Comment by niddal-imam** [comment]
I changed the batchSize, and it start printing.

Thanks
